app/views/posts/show.html.erb

```diff
@@ -1 +1,21 @@
-<h1>Here are the details of post 1 from user 1</h1>
\ No newline at end of file
+<article id="user-post">
+  <div class="post-wrapper">
+    <div class="post-header">
+      <div class="header">
+        <h1><%= @post.title %> by <%= @user.name %></h1>
+      </div>
+      <div>
+        <span>Comments: <%= @post.comments_counter %></span>
+        <span>Likes: <%= @post.likes_counter %></span>
+      </div>
+    </div>
+    <p style="post-content"><%= @post.text %></p>
+  </div>
+  <div class="comments-wrapper">
+    <% @post.recent_comments.each do |comment| %>
+      <div class="comment">
+        <span>User <%= comment.user_id %></span> : <span><%= comment.text %></span>
+      </div>
+    <% end %>
```

Comment:
- [ ] [Optional] Well! in this page, you are displaying recent comments for each post. However, as per the project requirements, you should show all the comments for a post on the post show page. For example, if a post has 14 comments, it is expected that the Post show page should display all 14 comments as per the wireframe. Please update your code accordingly to fulfill this requirement.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/simplegoose/blog_app/assets/55840999/0f739eba-7136-4fe2-8ddd-ac2666acf739) | ![image](https://github.com/simplegoose/blog_app/assets/55840999/f4fcc003-bb40-4056-a58d-5a72c3ad4754)|